dropwizard-testing/src/test/java/io/dropwizard/testing/app/PersonResource.java

```diff
@@ -32,6 +33,10 @@ public PersonResource(PeopleStore store) {
         this.store = store;
     }
 
+    public PersonResource() {
+        this.store = Mockito.mock(PeopleStore.class);
+    }
```

Comment:
Without this, the `PersonResourceExceptionMapperTest` is unable to instantiate a PersonResource, despite the bytebuddy enhanced binder that should be able to bind PersonResource to the constructed instance passed to the `ResourceTestRule`.  No idea why :(